guide.rst

```diff
@@ -259,41 +259,20 @@ without a selector and referred to from other messages, like the ``about``
 message above.
 
 
-Storing Additional Information
-==============================
-
-::
-
-    brand-name = Firefox
-        [gender] masculine
-
-    opened-new-window = { brand-name[gender] ->
-       *[masculine] { brand-name } otworzyl nowe okno.
-        [feminine] { brand-name } otworzyla nowe okno.
-    }
-
-Traits can be used to describe parameters of the entity that can be then
-used in other selectors.
-
-Imagine an entity ``brand-name`` that can be either *Firefox* or *Aurora*.  The 
-former is *masculine*, while the latter is *feminine*, so sentences that refer 
-to this entity may want to branch depending on the gender of it.
-
-
-HTML/XUL Attributes
-===================
+HTML Attributes
+===============
 
 ::
 
     login-input = Predefined value
-        [html/placeholder] example@email.com
-        [html/aria-label]  Login input value
-        [html/title]       Type your login email
+        .placeholder = example@email.com
+        .aria-label = Login input value
+        .title = Type your login email
```

Comment:
so, I just produced a new test FTL file using the new syntax. It's the closest thing we have from Firefox that resembles "big" FTL file and as such I believe it's valuable for us to test our syntax on it against our principles.

To remind principles:
 - the file should be readable without prior knowledge of FTL
 - the file should be easy to scan through
 - the file should be editable with minimal FTL knowledge

0.1 syntax: https://github.com/projectfluent/fluent-rs/blob/0.1/tests/workload-low.ftl
0.2 syntax: https://github.com/projectfluent/fluent-rs/blob/master/tests/workload-low.ftl

For one, I'd say we did manage to make it look cleaner (less sigils!)
But also, I believe the attribute alignment significantly helps with readability, so I'm not sure if I agree that preserving blame is worth it.